### ISSUE TITLE
MIREX model issue

### ISSUE THREAD

**Issue by Galvo87** [MTG/essentia#1255]
When mood tagging via MIREX model, I always get:
`TensorflowPredict: 'model/Sigmoid' is not a valid node name of this graph`
Any clue or code example for correctly linking this dataset?
The other ones are working fine.

**Comment by palonso** [comment]
Hi @Galvo87,
The name of the output node is available in the attached [metadata file](http://essentia.upf.edu/models/classifiers/moods_mirex/moods_mirex-musicnn-msd-1.json). This model was trained with a softmax activation layer, so set the parameter (`output="model/Softmax"`) to use it.
